src/Sulu/Bundle/MediaBundle/Media/ImageConverter/MediaImageExtractor.php

```diff
@@ -48,45 +47,37 @@ public function __construct(
         $this->videoThumbnail = $videoThumbnail;
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    public function extract($content)
+    public function extract($resource)
     {
-        $finfo = new \finfo();
-        $mimeType = $finfo->buffer($content, FILEINFO_MIME_TYPE);
+        $mimeType = mime_content_type($resource);
 
         if ('application/pdf' === $mimeType) {
-            return $this->convertPdfToImage($content);
+            return $this->convertPdfToImage($resource);
         }
 
         if ('image/vnd.adobe.photoshop' === $mimeType) {
-            return $this->convertPsdToImage($content);
-        }
-
-        if ('image/svg+xml' === $mimeType) {
```

Comment:
Any reason we don't convert SVGs anymore?

Reply:
@alexander-schranz says that this isnt neccesary anymore